roles/ceph-monitor/files/ceph_adjust.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
```

Comment:
We should make all of this code its own Python module that is installable via pip. (And then remove this line)

The reasons to do so are a few:
- if we ever pull in requirements from anything other than the standard library, we will want to manage their versions 
- making it its own module will allow us to create python console_script entrypoints...so a user can run it from the CLI, if necessary
- we will be able to update this code (ie. if we find a bug) without needing a whole new Ursula release.
- we can (and should) add unit tests...a module would let that happen.

I can help with making this happen.